src/libraries/System.Private.CoreLib/src/System/ArgumentOutOfRangeException.cs

```diff
@@ -22,31 +22,22 @@ public class ArgumentOutOfRangeException : ArgumentException
         // Creates a new ArgumentOutOfRangeException with its message
         // string set to a default message explaining an argument was out of range.
         public ArgumentOutOfRangeException()
-            : base(SR.Arg_ArgumentOutOfRangeException)
-        {
-            HResult = HResults.COR_E_ARGUMENTOUTOFRANGE;
-        }
+            : this(null, null, null) {}
 
         public ArgumentOutOfRangeException(string? paramName)
-            : base(SR.Arg_ArgumentOutOfRangeException, paramName)
-        {
-            HResult = HResults.COR_E_ARGUMENTOUTOFRANGE;
-        }
-
+            : this(paramName, null, null) {}
+ 
         public ArgumentOutOfRangeException(string? paramName, string? message)
-            : base(message, paramName)
-        {
-            HResult = HResults.COR_E_ARGUMENTOUTOFRANGE;
-        }
-
+            : this(paramName, null, message) {}
+ 
         public ArgumentOutOfRangeException(string? message, Exception? innerException)
```

Comment:
This is missing the same treatment.

Reply:
If there are other types where this would make sense, we can use this PR to do so.

Reply:
So there is no point constraining this to CoreLib.
This will affect all exception classes CoreLib (`ls -1 **/*Exception.cs | wc -l   # -> 100`) and some in other libs
i.e. 
https://github.com/dotnet/runtime/blob/3b21b909f47ee07ec7f4cad0edde5404ebd85dd2/src/libraries/System.Data.Common/src/System/Data/DataException.cs#L26-L31
https://github.com/dotnet/runtime/blob/3b21b909f47ee07ec7f4cad0edde5404ebd85dd2/src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/IdentityNotMappedException.cs#L20-L31
117 classes, but not every Exception out here have custom message

----

Are you ok with proposed pattern with constructor delegation?

Reply:
> Are you ok with proposed pattern with constructor delegation?

I'd prefer we just add `?? SR.Whatever` where each message is passed along to the base.

Delegating from ctor to ctor unnecessarily keeps all larger constructors rooted and prevents them from being trimmed away when the smaller one is being used. It's also a lot more churn.